x11-wm/mutter/mutter-9999.ebuild

```diff
@@ -58,7 +58,7 @@ RDEPEND="
 	>=media-libs/graphene-1.10.2[introspection?]
 	x11-libs/gdk-pixbuf:2
 	>=x11-libs/pango-1.46[introspection?]
-	>=x11-libs/cairo-1.14[X]
+	>=x11-libs/cairo-1.14
```

Comment:
Are we sure this doesn't affect cairo beyond providing extra libraries, in a way that it would rely on for x11 or xwayland client?